### ISSUE TITLE
CodeMirror fails to load on database page

### ISSUE THREAD

**Issue by simonw** [simonw/datasette#560]
It's not loading on https://latest.datasette.io/fixtures

But it does load on https://latest.datasette.io/fixtures?sql=select+*+from+facetable

**Comment by simonw** [comment]
```
TypeError: textarea is null[Learn More] codemirror-5.31.0.js:9508:3
    fromTextArea http://127.0.0.1:8001/-/static/codemirror-5.31.0.js:9508
    <anonymous> http://127.0.0.1:8001/fixtures:286
```

**Comment by simonw** [comment]
Here's why:

https://github.com/simonw/datasette/blob/6abe6faff6b035e9334dd05f8c741ae9b7a47440/datasette/templates/_codemirror_foot.html#L2

This works on the `query.html` template:

https://github.com/simonw/datasette/blob/6abe6faff6b035e9334dd05f8c741ae9b7a47440/datasette/templates/query.html#L38

But fails on the `database.html` template:

https://github.com/simonw/datasette/blob/6abe6faff6b035e9334dd05f8c741ae9b7a47440/datasette/templates/database.html#L28

**Comment by simonw** [comment]
This is released in 0.29.3

Compare https://v0-29.datasette.io/fixtures (broken) with https://v0-29-3.datasette.io/fixtures (fixed)
